static-ct-api.md

```diff
@@ -282,7 +282,9 @@ opaque Fingerprint[32];
 “timestamped_entry” is the `TimestampedEntry` sub-structure of a
 `MerkleTreeLeaf` according to RFC 6962, Section 3.4.
 
-“pre_certificate” is the Precertificate submitted for auditing.
+“pre_certificate” is the Precertificate submitted for auditing, corresponding
+to the `pre_certificate` specified in RFC 6962, Section 3.1. It is not the same
+as the PreCert in `timestamped_entry`, which is a TBS certificate.
```

Comment:
```suggestion
to the `pre_certificate` specified in RFC 6962, Section 3.1. It is not the same
as `timestamped_entry.signed_entry`, which is a PreCert containing a TBSCertificate.
```

Reply:
Minor nit: the timestamped_entry.signed_entry PreCert is issuer hash plus TBSCertificate.